apiary_v2.apib

```diff
@@ -1131,7 +1237,32 @@ This endpoint can be used to retrieve a paginated list of the individual items i
     + Body
 
           {
-          }
+            "count": 12,
+            "next": "https://partner.embed.unmade.com/v2/orders/123456/items?page=2",
+            "previous": null,
+            "results": [
+              {
+                "design": "fd792895-1e32-41b1-90af-683f0cf9f753",
+                "quantity": 1,
+                "partner_data": {
+                  "UNMADE_SOURCE_DESIGN_ID": "fd792895-1e32-41b1-90af-683f0cf9f753",
+                  "UNMADE_ORDER_ITEM_DESIGN_ID": "fd792895-1e32-41b1-90af-683f0cf9f753"
```

Comment:
maybe add here one of the "example" ones they could send so they know it can have more things than this 2 that we add automatically. 

Maybe we need to explain the 2 we add automatically?

Reply:
I need to work out what source actually means here

Reply:
source means the editor design so either parent or grand parent if it's a roster (if I remember well)